### DISCUSSION TITLE
Jupyter Notebook in PyCharm

### DISCUSSION THREAD

**Discussion by alexander0042** [community/community#7639] [Copilot Conversations] [answered]
Hi,

I got copilot installed successfully in PyCharm, and it's working incredibly well in regular .py files. 
However, when I try to open a Jupyter notebook, I'm consistently getting a: `java.lang.Throwable: Temporary file is invalid or physical: Jupyter Notebook` error. 

I've updated Copilot and PyCharm to the latest versions, so that's not it, and was wondering if this was a known issue or if there were any suggestions here?

**Comment by jansorg** [comment, marked as answer]
Thanks! That's a known issue and it will be fixed with the next update.

**Reply by AleksandrovichK** [reply]
Well, it wasn't, apparently. It's been a year lol
